composer.json

```diff
@@ -35,16 +52,15 @@
 
     "roave/security-advisories": "dev-master",
 
-    "behat/behat": "^3.5",
-    "behat/mink-extension": "^2.3",
-    "behat/mink-browserkit-driver": "^1.3",
-    "friends-of-behat/symfony-extension": "^2.0",
+    "behat/behat": "dev-master as 3.6",
+    "behat/mink-extension": "dev-patch-4",
+    "behat/mink-browserkit-driver": "dev-symfony-5",
+    "friends-of-behat/symfony-extension": "2.1.0-BETA.1",
 
     "phpunit/phpunit": "^8.2",
     "mockery/mockery": "^1.2",
 
-    "fzaninotto/faker": "^1.8",
-    "wouterj/eloquent-bundle": "^1.0"
```

Comment:
They don't have Symfony 5 support (https://github.com/wouterj/WouterJEloquentBundle) and as this only for 1 example... 🤗